daffodil-cli/src/windows/Product_en-us.wxl

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The above LICENSE.rtf file should be deleted since that is now autogenerated.

Reply:
Done.